lib/internal/fs/read_file_context.js

```diff
@@ -73,9 +72,10 @@ class ReadFileContext {
     let length;
 
     if (this.size === 0) {
-      buffer = this.buffer = Buffer.allocUnsafeSlow(kReadFileBufferLength);
+      buffer = Buffer.allocUnsafeSlow(kReadFileUnknownBufferLength);
+      this.buffer = buffer;
```

Comment:
minor nit: maybe this could be moved to after `length = ` to more closely match the order of assignments in the `else`?